components/hab/src/main.rs

```diff
@@ -486,6 +497,61 @@ fn sub_origin_transfer_ownership(ui: &mut UI, m: &ArgMatches<'_>) -> Result<()>
     command::origin::transfer::start(ui, &url, &token, &origin, &account)
 }
 
+fn sub_accept_origin_invitation(ui: &mut UI, m: &ArgMatches<'_>) -> Result<()> {
+    let origin = m.value_of("ORIGIN").expect("required ORIGIN");
+    let invitation_id: i64 = m.value_of("INVITATION_ID")
+                              .expect("required INVITATION_ID")
+                              .parse()
+                              .expect("INVITATION_ID PARSE");
+    let url = bldr_url_from_matches(&m)?;
+    let token = auth_token_param_or_env(&m)?;
+    command::origin::invitations::accept::start(ui, &url, &token, &origin, invitation_id)
+}
+
+fn sub_ignore_origin_invitation(ui: &mut UI, m: &ArgMatches<'_>) -> Result<()> {
+    let origin = m.value_of("ORIGIN").expect("required ORIGIN");
+    let invitation_id: i64 = m.value_of("INVITATION_ID")
+                              .expect("required INVITATION_ID")
+                              .parse()
+                              .expect("INVITATION_ID PARSE");
```

Comment:
Same observation here, and elsewhere where appropriate.

Reply:
Response in https://github.com/habitat-sh/habitat/pull/7349#discussion_r368139365